lc-hax/Scripts/Helpers/FindNonNullObjectsOfType.cs

```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+namespace Hax;
+
+public static partial class Helper {
+    public static IEnumerable<T> FindNonNullObjectsOfType<T>() where T : Object => Object.FindObjectsOfType<T>().Where(obj => obj != null);
+}
```

Comment:
This is not necessary. Unity's `FindObjectsOfType` already searches for non-null objects. Both this and Unity's implementation, however, is not able to check for `null` that occur after finding the objects.

Reply:
> `FindObjectsOfType` already searches for non-null objects

that's definitely not what the 150 MB of `NullReferenceException` errors in BepInEx logs seem to imply

Reply:
When I use `FindObjectsOfType` instead of my `FindNonNullObjectsOfType`, I get a crap ton of these on end of round:

    [Error  : Unity Log] NullReferenceException
    Stack trace:
    UnityEngine.Component.GetComponent[T] () (at <e27997765c1848b09d8073e5d642717a>:IL_0021)
    Hax.ESPMod+<>c.<InitialiseRenderers>b__25_1 (Landmine landmine) (at <407cb0d5d3654832bcdc8b326a3493dc>:IL_0000)
    System.Linq.Enumerable+SelectArrayIterator`2[TSource,TResult].MoveNext () (at <dab7f68612224ba3ae40f651d44f9d4c>:IL_003A)
    Extensions.ForEach[T] (System.Collections.Generic.IEnumerable`1[T] array, System.Action`1[T] action) (at <407cb0d5d3654832bcdc8b326a3493dc>:IL_0017)
    Hax.ESPMod.OnGUI () (at <407cb0d5d3654832bcdc8b326a3493dc>:IL_0045)

Reply:
Okay, I understand why. It's because `IEnumerable` is lazy-loaded. This means, by the time I use the renderers, it is possible for them to be null. In that case, we still don't need a dedicated helper since this is a very special case. Instead, let's put this method within the ESP class.

Reply:
I'll make the changes.